src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs

```diff
@@ -99,6 +99,7 @@ private void InitializeNavigationViewItems()
 				AddMenuItem<ExpandingBottomSheetSamplePage>(icon: "ExpandingBottomSheetIcon");
 				AddMenuItem<FabSamplePage>(content: "FAB", icon: "FabIcon");
 				AddMenuItem<NavigationViewSamplePage>(icon: "NavigationViewIcon");
+				AddMenuItem<HyperlinkButtonSamplePage>(icon: "ButtonIcon");
```

Comment:
Should we even try to order them alphabetically?

Reply:
Of course we should!